frontend/src/App.tsx

```diff
@@ -17,7 +17,7 @@ function App() {
     <div className="app-container">
       <Routes>
         <Route path="/" element={<Layout />}>
-          <Route index element={<Candidates />} />
+          <Route index element={<ThankyouForApplying />} />
```

Comment:
Keep the index element as Candidates
and then add a new route path for the thankyou page.